src/verifier/tdx/mod.rs

```
@@ -0,0 +1,164 @@
+
+            ccel.integrity_check(rtmr_from_quote)?;
+        }
+        None => {
+            warn!("There is no CC EventLog in Evidence!!!");
+        }
+    }
+
+    // Return Evidence parsed claim
+    generate_parsed_claim(quote, ccel_option)
+}
+
+fn generate_parsed_claim(
+    quote: Quote,
+    cc_eventlog: Option<CcEventLog>,
+) -> Result<TeeEvidenceParsedClaim> {
+    // Current Parsed Claim just a example
+    // TODO: Claim key name shall be consistent with RVPS
+
+    let mut claim_map = Map::new();
+
+    // Claims from TD Quote.
+    claim_map.insert(
+        "tdx-tcb-svn".to_string(),
+        serde_json::Value::String(hex::encode(quote.report_body.tcb_svn)),
+    );
+    claim_map.insert(
+        "tdx-mrseam".to_string(),
+        serde_json::Value::String(hex::encode(quote.report_body.mr_seam)),
+    );
+    claim_map.insert(
+        "tdx-mrtd".to_string(),
+        serde_json::Value::String(hex::encode(quote.report_body.mr_td)),
+    );
+    claim_map.insert(
+        "tdx-mrconfigid".to_string(),
+        serde_json::Value::String(hex::encode(quote.report_body.mr_config_id)),
+    );
+
+    // Claims from CC EventLog.
+    match cc_eventlog {
+        Some(ccel) => {
+            match ccel.query_digest(MeasuredEntity::TdPayload) {
+                Some(td_paylod_digest) => {
+                    claim_map.insert(
+                        "tdx-kernel".to_string(),
+                        serde_json::Value::String(td_paylod_digest),
+                    );
+                }
+                None => todo!(),
+            }
+            match ccel.query_digest(MeasuredEntity::TdPayloadInfo) {
+                Some(td_paylod_info_digest) => {
+                    claim_map.insert(
+                        "tdx-kernel-params".to_string(),
+                        serde_json::Value::String(td_paylod_info_digest),
+                    );
+                }
+                None => todo!(),
+            }
+        }
+        None => todo!(),
```

Do we need to report an error here?